### PR TITLE
Project Addition: added react project

### PR DESCRIPTION
**Pull request by itsme-shivamkumar**
Project Addition for Issue no. #471 
# Desume Editor
## Features
- Google authentication for secure sign-in
- Editing of resumes using HTML and CSS directly in the app
- Ability to add styling like font size, letter spacing, font weight, color, and opacity
- Option to add elements using pure HTML code and inline style
- Option to delete any div in the HTML-based resume
- Responsive web mode for desktop screens
- Framer Motion library for animations
- jsPDF and html2canvas libraries for generating and downloading PDF resumes
- Installation
- To use Desume Editor, simply visit the website and sign in using your Google account.

## Usage
- Sign in with your Google account.
- Edit your resume using the text editor and styling options.
- Save your changes and download your resume in HTML or PDF format.
## Future Development
- Addition of layout customization and animation features
- Responsive design for mobile and tablet screens
- Technologies Used
## Desume Editor was built using the following technologies:
- React
- HTML
- CSS
- JavaScript
- Google authentication
- Framer Motion library
- jsPDF library
- html2canvas library

## Contributing
This project was developed by a single developer [itsme-shivamkumar](https://github.com/itsme-shivamkumar), but contributions are always welcome. Please submit a pull request for any changes or fixes.

### CODE REVIEW

**Comment by priyankarpal** [comment]
without the issue number, we are unable to merge this PR.

Check the [Contributing.md](https://github.com/priyankarpal/ProjectsHut/blob/main/contributing.md)

**Comment by itsme-shivamkumar** [comment]
> without the issue number, we are unable to merge this PR.
> 
> Check the [Contributing.md](https://github.com/priyankarpal/ProjectsHut/blob/main/contributing.md)

I have added the issue number. Kindly merge it, and from next time I'll try not to repeat this. Thank You❣️

**Comment by priyankarpal** [comment]
there is a conflict error. Please resolve it

**Comment by itsme-shivamkumar** [comment]
> there is a conflict error. Please resolve it

I have resolved it, Kindly Review.

File: src/DB/projects.json [hunk 1]
```diff
@@ -691,20 +691,20 @@
...
     "Projects": [
       {
-        "link": "https://vaibhav-maurya.netlify.app/",
-        "title": "Portfolio site",
-        "description": "A portfolio project showcases an individual's or a professional's work, skills, and accomplishments in a concise and visually appealing manner to demonstrate their expertise and attract potential opportunities.",
-        "tech": ["CSS", "JavaScript", "HTML"]
+        "link": "https://github.com/itsme-shivamkumar/desume",
+        "title": "Desume | Resume Builder For Developers",
+        "description": "Desume Editor is a React-based web app that allows developers to edit their resumes with ease. With Desume Editor, developers can create or modify resumes with HTML and CSS directly in the app.",
+        "tech": ["React", "Firebase", "JavaScript", "CSS", "HTML", "Framer Motion"]
```
Comment (priyankarpal) [inline src/DB/projects.json:707]: you've deleted someone's contribution. Don't do this. 

You can close this PR & add create one

Comment (itsme-shivamkumar) [reply src/DB/projects.json:707]: I didn't knew it that I couldn't delete that, Sure, I'll close this and add new one

**Pull request closed without merging**